tool/flutter_ci_script_stable.sh

```diff
@@ -15,6 +15,7 @@ declare -ar PROJECT_NAMES=(
     "add_to_app/prebuilt_module/flutter_module"
     "android_splash_screen"
     "animations"
+    "code_sharing"
```

Comment:
@domesticmouse - After this commit, I was expecting to see my new sample audited in the CI pass, yet I'm not seeing it. Do you know if I forgot to make a change when adding this?

Reply:
This probably should be `"code_sharing/client"` and `"code_sharing/server"`

Reply:
omg, duh